### ISSUE TITLE
Convergence on a custom dataset 

### ISSUE THREAD

**Issue by VladimirYugay** [shariqfarooq123/AdaBins#32]
Hey there,

Thanks for your work. 

I'm currently trying to overfit for only one image with the basic setup. 
However, after 100 epochs with standard parameters, the losses do not really converge during training. For example, SILog loss is plateaued during training at 1.5, and chamfer loss is around 1.1. See the wdb board [here](https://wandb.ai/xokage/MDE-AdaBins-motsynth/runs/2aqbqttd?workspace=user-xokage).

Moreover, as you can see from the dashboards, the convergence seems really slow for just one image. Are there any hints to improve specifically for AdaBins losses? (Already tried various lr and turning off the scheduler) 

Regarding the input. My depth map labels are actually numpy arrays with real depth values ranging from 0 to 80 meters. 

![result](https://user-images.githubusercontent.com/12258972/120552533-a8b98100-c3f7-11eb-9a85-59a212b06914.png)

Any hints on how to make the network converge to a single image would be appreciated!

**Comment by Saurav-31** [comment]
@VladimirYugay   anything that you found useful? Can you please share here.

**Comment by N00Bzzsd** [comment]
hi 
I have some problems in training custom dataset by this model.
 it obviously that you did the work greatly！
Can you tell me how to  training custom dataset by this model？
 thanks！

**Comment by VladimirYugay** [comment]
I implemented the custom dataset [here](https://github.com/VladimirYugay/AdaBins) and it worked fine on that synthetic dataset.

**Comment by N00Bzzsd** [comment]
> I implemented the custom dataset [here](https://github.com/VladimirYugay/AdaBins) and it worked fine on that synthetic dataset.

wow, it is quite useful for my work. thanks!!!
